.github/workflows/check-changelog.yml

```diff
@@ -0,0 +1,20 @@
+name: check-changelog
+
+on:
+  pull_request:
+    types: [opened, synchronize, labeled, unlabeled]
```

Comment:
Will this rerun if a PR is updated? i.e. if someone pushes more commits to an open PR?

Reply:
Yes!